### ISSUE TITLE
Integration Test Failure

### ISSUE THREAD

**Issue by amouat** [estesp/manifest-tool#88]
When running the integration tests, I get:

```
Manifest entry for image trow.test:8443/test/aarch64_alpine:latest has unsupported os/arch or os/arch/variant combination: linux/arm64/armv8
```

This seems to be caused by this line in docker/util.go https://github.com/estesp/manifest-tool/blob/master/docker/util.go#L25

```
"linux/arm64/v8":  true,
```

(Note armv8 vs v8).

Changing this line in the code makes the tests pass for the Trow registry (on master). I'm not sure if that's the correct fix or if something else is wrong.

**Comment by estesp** [comment]
Do you know what build tool created the image that has a variant of `armv8`? While I'm not sure we have definitive guidance on `variant`, for ARM the commonly accepted values across the runtimes I know about are `v3` - `v8` without any of them accepting a pre-pending "arm" to the variant name. 

The **platforms** package from containerd, for example, is used across several other projects as the definitive platform OS/arch/variant generator, and the ARM usage is detailed in [this file](https://github.com/containerd/containerd/blob/master/platforms/cpuinfo.go)

**Comment by amouat** [comment]
I'll have a look. Sounds like the right fix is somewhere else though.

Just to give you some background, I added support for the manifest list content type to Trow, which I thought would be enough to support `docker manifest create/push`, but it gives me strange errors. So I tried with manifest-tool, which seems fine except for this problem.

It's off-topic for this repo, but do you know if I'm meant to support any new API methods for manifest lists? I think I'll need to dig into the docker code to figure out what's going on.

**Comment by amouat** [comment]
So I looked into it a little more and I don't think I was clear in my initial report.

I'm running the integration test in this repo: https://github.com/estesp/manifest-tool/blob/master/integration/test-registry.sh

And I'm getting the above failure. It _seems_ to be caused the `test-registry.yaml` file: https://github.com/estesp/manifest-tool/blob/master/integration/test-registry.yml#L23 which appears to disagree with the code. 

I probably should have fixed it by changing the test code instead instead of the main codebase, but I just grepped for the error and worked from there.
